__tests__/test.js

```diff
@@ -1,21 +1,81 @@
 const { expect } = require('chai');
+const sinon = require('sinon');
 const {
     createIdObject,
     fetchPageOfDataAndFilter,
     fetchUserDataAndAddToCSV,
     filterContributorByTime,
     filterResponseForImportantEvents,
     getOrThrow,
-    parseDatesFromArgv
+    parseDatesFromArgv,
 } = require('../index');
 
 const envBeforeChanges = Object.assign({}, process.env);
+
 beforeEach(() => {
     process.env = Object.assign({}, envBeforeChanges);
 });
 after(() => {
     process.env = envBeforeChanges;
 });
+/* eslint-disable camelcase */
```

Comment:
Generally I try to avoid lint overrides in the code. Is this one required because the externally-specified field names are snake_case?